src/main/resources/application.properties

```diff
@@ -1 +1,12 @@
+jdbc.driver=com.mysql.jdbc.Driver
+jdbc.url=jdbc:mysql://localhost:3306/dev_db
+jdbc.username=root
+jdbc.password=root
 
+
+url.prefix = GET/weather-conditions" + "\\?" + "location=
+url.tomorrowURL = https://api.tomorrow.io/v4/timelines?location=
+url.apiKey = apikey=fvFU2JQBMP1QX7MeW44ghUQnNiy97uX6
+url.fields = fields=temperature,humidity,windSpeed,rainIntensity
```

Comment:
See my previous comment about having a single source of truth for the supported fields. Think how to design your code s.t. adding a field or an operator will be easiest, and having a single source of truth for supported fields and operators.